mtv-plan/application.properties

```diff
@@ -1,3 +1,5 @@
+quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator/api/notifications}
```

Comment:
same: with https://github.com/parodos-dev/serverless-workflows/pull/307 there is a path in the openapi spec, the URL shall not contain the path; the default value should be http://backstage-backstage.rhdh-operator